lib/helpers/logger.js

```diff
@@ -0,0 +1,20 @@
+const blgr = require('blgr');
+
+// TODO: default to info level
+const LOG_LEVEL = process.env.LOG_LEVEL || 'debug';
+const logger = new blgr(LOG_LEVEL);
+
+// set different contexts here
+logger.context('actions');
+logger.context('reducers');
+
+logger.open();
+
```

Comment:
I'm think that we create a logger in `bpanel-utils` that can be imported into all of the projects.
It would be a class that wraps `blgr` but gives some helpers around creating and getting contexts and it would also automatically attach a `setLogLevel` method on the `window` if the `window` exists. If a nodejs environment, it would expose the `setLogLevel` behind an `http` endpoint, but we don't need that functionality right now

Reply:
Just re-mentioning my other comment that I don't really feel like we need this. The redux devtools does a lot of this with a lot more information and functionality without cluttering the console at all. I also think a middleware makes way more sense for something like this rather than adding a log statement into each and every action and reducer. There are things we will want to notify users about but those will use some kind of notification API.

Reply:
I'd be curious to see the situations where this provides extra utility over the existing devtools and temporary debug statements. I generally feel that it's worth thinking about why certain conventions exist before considering doing something different especially if it means adding additional tooling. Anyway, this isn't a huge deal so not worth dwelling on anymore I don't think. We can think of what a more robust devtool system might look like later.